userspace/libsinsp/sinsp.h

```diff
@@ -892,6 +892,22 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 	*/
 	std::unordered_set<uint32_t> enforce_sys_ppm_sc_set(std::unordered_set<uint32_t> ppm_sc_set = {});
 
+	/*!
+	  \brief Get all the available ppm_sc.
+	  Does enforce minimum sinsp state set.
+	*/
+	std::unordered_set<uint32_t> get_all_ppm_sc();
+
+	/*!
+	  \brief Get the name of all the ppm_sc provided in the set.
+	*/
+	std::unordered_set<std::string> get_syscalls_names(const std::unordered_set<uint32_t>& ppm_sc_set);
```

Comment:
`get_ppm_sc_names`?

Reply:
Yes and no, these are the names of the real syscalls so i don't know what to do here :thinking: